### PR TITLE
Default font weight

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: ui/src/styles.css [hunk 1]
```diff
@@ -90,3 +89,154 @@ input[type=text]:focus, input[type=password]:focus {
 .noUi-connect {
   background: rgb(92, 183, 225);
 }
+
+/*
+ * Override clarity fonts
+ */
+body {
```
Comment (johnstill) [inline ui/src/styles.css]: so FYI, if the rule is the same for all these selectors, they can just be combined with commas.  I.e.

```
.login-wrapper .login .title,
.login-wrapper .login .trademark,
.modal-header .modal-title,
/* etc */
h1, h2, h3, h4, h5, h6 {
  /* font-family rule or font-weight rule or what have you */
}
```

That might help clean this up a little bit.

**Comment by blrubenstein** [comment]
I would use `body *` but with us doing font-weight as well, and not just font-family, it gets a little less clean.

**Comment by blrubenstein** [comment]
Should be good! @johnstill PTAL

**Comment by johnstill** [comment]
Ugh. Ok... we'll just add to / remove from / tweak this as necessary I suppose.

**Pull request merged**